guides/v2.2/cloud/live/go-live-checklist.md

```diff
@@ -86,5 +86,5 @@ You can also test using the following 3rd party options:
 * [Blackfire]({{ page.baseurl }}/cloud/project/project-integrate-blackfire.html) (provided): Helps track through the issues New Relic finds and helps you dig deeper into the issue for specifics. Blackfire profiles the environment and helps locate bottlenecks in depth: process, method call, query, load, and so on.
 * [WebPageTest](https://www.webpagetest.org/) and [Pingdom](https://www.pingdom.com/): Real-time analysis of your site pages load time with different origin locations. Pingdom may cost a fee. WebPageTest is a free tool.
 
-#### Next step:
+**Next step:**
```

Comment:
It's a shame you have to do this manually. Out of scope for this PR, but you might want to consider leveraging the layout for tutorials to solve for the "previous" and "next" on page navigation.

Reply:
Yes, you're right. I think these topics will go away, though. I didn't want them to break the MD001 effort. Cloud doesn't have many of these left.